Extensions/GoogleOptimize/Src/Tasks/GoogleOptimize/task.json

```diff
@@ -0,0 +1,166 @@
+{
+   "id":"e7307da0-85f4-11e9-9dec-fdd4f9b653a3",
+   "name":"GoogleOptimize",
+   "friendlyName":"Google Optimize",
+   "description":"Azure devops extension for google optimize and content experiments",
+   "author":"Microsoft Corporation",
+   "helpMarkDown":"[More Information](https://github.com/Microsoft/azure-pipelines-extensions/blob/master/Extensions/GoogleOptimize/Src/README.md)",
+   "category":"Utility",
+   "visibility":[
+      "Build",
+      "Release"
+   ],
+   "demands":[
+
+   ],
+   "version":{
+      "Major":"1",
+      "Minor":"0",
+      "Patch":"0"
+   },
+   "preview": "true",
+   "minimumAgentVersion":"1.95.0",
+   "instanceNameFormat":"GoogleOptimize $(message)",
+   "inputs":[
+      {
+         "name":"googleEndpoint",
+         "type":"connectedService:googleoptimize",
+         "label":"Google Optimize Service Connection",
+         "defaultValue":"",
+         "required":true,
+         "helpMarkDown":"Select the Optimize service endpoint to be used to be used to manage A/B test"
+      },
+      {
+         "name":"accountId",
+         "type":"string",
+         "label":"Account Id",
+         "defaultValue":"",
+         "required":true,
+         "helpMarkDown":"Enter the account Id which can be found in account settings inside admin tab"
+      },
+      {
+         "name":"webPropertyId",
+         "type":"string",
+         "label":"Web Property Id",
+         "defaultValue":"UA-XXXXX-Y",
+         "required":true,
+         "helpMarkDown":"Enter the web Property Id which can be found in property settings inside admin tab. Type- UA-XXXXX-Y"
```

Comment:
would giving a link be more useful, https://support.google.com/analytics/answer/1042508

Reply:
The comment to provide link will be applicable to other relevant optimize or analytics properties that you take as input below.